thrifty-schema/src/test/java/com/microsoft/thrifty/schema/LoaderTest.java

```diff
@@ -358,6 +359,35 @@ public void canLoadAndLinkOfficialTestThrift() throws Exception {
         loader.load();
     }
 
+    @Test
+    public void includesWithRelativePaths() throws Exception {
+        tempDir.newFolder("foo");
+        File f1 = tempDir.newFile("a.thrift");
+        File f2 = tempDir.newFile(Joiner.on(File.separatorChar).join("foo", "b.thrift"));
+
+        String a = "namespace java com.microsoft.thrifty.test.includesWithRelativePaths\n" +
+                "\n" +
+                "enum A {\n" +
+                "  ONE, TWO, THREE\n" +
+                "}";
+
+        String b = "include '" + f1.getCanonicalPath() + "'\n" +
```

Comment:
I don't think this tests what you want it to; did you mean to say `include '../a.thrift'`?